src/vs/workbench/parts/html/browser/webview.html

```diff
@@ -9,26 +9,41 @@
 	<script>
 		'use strict';
 
+		var initData = {};
+
+		function styleBody(body) {
+			if (body) {
+				let currentClasses = body.className.split(' ');
+				let strippedClasses = currentClasses.filter((cl) => {
+					return !initData.bodyClasses.remove.includes(cl);
+				});
+				body.className= `${strippedClasses.join(' ')} ${initData.bodyClasses.add}`;
+			}
```

Comment:
Could we just use: `body.classList.remove("vs", "vs-dark", "hc-dark"); body.classList.add("theActualTheme")`?